tests/testGetEimage.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function
+#
+# LSST Data Management System
+# Copyright 2008-2013 LSST Corporation.
```

Comment:
I guess, until we update to the "new" short boilerplate, we should at least keep the copyright date correct.